apps/src/applab/designElements/photochooser.jsx

```diff
@@ -0,0 +1,226 @@
+import $ from 'jquery';
+import PropTypes from 'prop-types';
+import React from 'react';
+import PropertyRow from './PropertyRow';
+import BooleanPropertyRow from './BooleanPropertyRow';
+import ColorPickerPropertyRow from './ColorPickerPropertyRow';
+import ZOrderRow from './ZOrderRow';
+import EventHeaderRow from './EventHeaderRow';
+import EventRow from './EventRow';
+import BorderProperties from './BorderProperties';
+import themeValues from '../themeValues';
+import {ICON_PREFIX_REGEX} from '../constants';
+import * as elementUtils from './elementUtils';
+import * as assetPrefix from '../../assetManagement/assetPrefix';
```

Comment:
NIT: (up to you) For elements with more than one level of nesting, I generally use @cdo notation. i.e. this would be `@cdo/apps/assetManagement/assetPrefix`